### ISSUE TITLE
new .tile classes are missing from npm build?

### ISSUE THREAD

**Issue by SimulatedGREG** [jgthms/bulma#170]
Using bulma@0.0.19 from npm.

`css/bulma.css`, nor `css/bulma.min.css`contain new .tile classes when installing from npm.
Tried `npm install bulma@latest`, just to be sure and am still not finding the new .tile classes.
They can be found in the source sass files though, so I'll just build from there for the time being.

`https://cdnjs.cloudflare.com/ajax/libs/bulma/0.0.19/css/bulma.min.css` seems to be missing them as well.

Also I found this class inside `bulma/components/grid.sass` in both this github repo and the npm package, which I'm not sure is a legitimate class or not, but I thought I should point it out.
```sass
.tilefiejsoif
  flex: 1
  // Modifiers
  &.is-parent
    padding: 10px
...
```

Anyways, thanks for such an amazing CSS library. Keep up the good work.

**Comment by jackel27** [comment]
I do have a workaround for this. Using SASS or CSS, put in your style and you should be good to go. Seems to work fine for me on codepen and local dev. (Did not include the tablet portion?)

Have not fully tested, but everything I tried so far seems to display properly.

## css:
```css
.tile {
  align-items: stretch;
  display: flex;
  flex: 1;
}
.tile.is-ancestor {
  margin-left: -10px;
  margin-right: -10px;
  margin-top: -10px;
}
.tile.is-ancestor:last-child {
  margin-bottom: -10px;
}
.tile.is-ancestor:not(:last-child) {
  margin-bottom: 10px;
}
.tile.is-child {
  margin: 0;
  display: inline-block;
}
.tile.is-parent {
  padding: 10px;
}
.tile.is-vertical {
  flex-direction: column;
}
.tile.is-vertical > .tile.is-child:not(:last-child) {
  margin-bottom: 20px;
}
```
## sass:
```sass
.tile
  align-items: stretch
  display: flex
  flex: 1
  &.is-ancestor
    margin-left: -10px 
    margin-right: -10px 
    margin-top: -10px 
    &:last-child
      margin-bottom: -10px
    &:not(:last-child)
      margin-bottom: 10px
  &.is-child
    margin: 0 
    display: inline-block
  &.is-parent
    padding: 10px 
  &.is-vertical
    flex-direction: column
    & > .tile.is-child:not(:last-child)
      margin-bottom: 20px 
```

Loving this! 👍

**Comment by jgthms** [comment]
It's been fixed in 0.0.20

**Comment by infacq** [comment]
still missing in css
